src/fundamental/confidential-virt-fundamental.h

```diff
@@ -0,0 +1,70 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+/* Keep CVM detection logic in this file at feature parity with
+ * that in src/efi/boot/vmm.c */
+
+#define CPUID_PROCESSOR_INFO_AND_FEATURE_BITS UINT32_C(0x1)
```

Comment:
This header should include `stdint.h`

Reply:
It wasn't clear to me that this is actually needed, as GCC appears to provide UINT32_C definition already.

Reply:
Oh yes, I missed that `stdint.h` was getting pulled in indirectly from another header. Will address this.